frontend/src/components/editProject/EditCollaboratingOrganizations.js

```diff
@@ -0,0 +1,53 @@
+import { List } from "@material-ui/core";
+import React from "react";
+
+import MiniOrganizationPreview from "../organization/MiniOrganizationPreview";
+import AutoCompleteSearchBar from "../search/AutoCompleteSearchBar";
+
+export default function EditCollaboratingOrganizations({
+  texts,
+  baseUrl,
+  searchBarClass,
+  handleAddCollaboratingOrg,
+  organizationPreviewsContainerClass,
+  organizationPreviewClass,
+  selectedCollaboratingOrgs,
+  handleDeleteCollaboratingOrg,
```

Comment:
Nit: given these are net new, do you mind alphabetizing them for maintainability? 

VS Code (if that's your primary editor) makes this really easy to do from the command palette (let me know if you have questions on how to use!).

Reply:
I wasn't aware of this feature, thanks for sharing! :)